### PR TITLE
chage: add page

### PR DESCRIPTION
**Pull request by slash3b**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

File: pages/linux/chage.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# chage
+
+> Change user account and password expiry information.
+
+- List password information for the user:
+
+`chage -l {{user_name}}`
+
+- Enable password expiration:
```
Comment (agnivade) [inline pages/linux/chage.md]: I think you can combine this and the next example with - 
```
Set the number of days to be 10, after which the user has to change the password

sudo chage -m {{10}} {{user_name}}
```

File: pages/linux/chage.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Disable password expiration:
+
+`sudo chage -M 99999 {{user_name}}`
+
+- Set account expiration date:
+
+`sudo chage -E {{YYYY-MM-DD}}`
+
+- Force user to chage password on next log in:
```
Comment (agnivade) [inline pages/linux/chage.md]: change the password ?

Comment (agnivade) [inline pages/linux/chage.md]: change the password ?

Comment (agnivade) [inline pages/linux/chage.md]: change the password ?

File: pages/linux/chage.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`sudo chage -d 0`
```
Comment (agnivade) [inline pages/linux/chage.md]: 0 in tokens (`{{}}`).

File: pages/linux/chage.md (hunk 2 above)
Comment (slash3b) [reply pages/linux/chage.md]: sorry for typo,
thanks

File: pages/linux/chage.md (hunk 3 above)
Comment (slash3b) [reply pages/linux/chage.md]: do we need to wrap it in curly braces here ?
I mean it is like that `-M 99999` which is required explicitly.

File: pages/linux/chage.md (hunk 1 above)
Comment (slash3b) [reply pages/linux/chage.md]: could you please check the fixed version?

File: pages/linux/chage.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+`chage -l {{user_name}}`
+
+- Enable password expiration in 10 days:
+
+`sudo chage -M {{10}} {{user_name}}`
+
+- Disable password expiration:
+
+`sudo chage -M 99999 {{user_name}}`
```
Comment (agnivade) [inline pages/linux/chage.md]: The man page says that `Passing the number -1 as MAX_DAYS will remove checking a password's validity`. So, shouldn't it be -1 ?

Comment (slash3b) [reply pages/linux/chage.md]: you are absolutely right, and I'll change that.
```
➜  ~ sudo chage -l test
Last password change                                    : Dec 05, 2017
Password expires                                        : never
Password inactive                                       : never
Account expires                                         : never
Minimum number of days between password change          : 0
Maximum number of days between password change          : 99999
Number of days of warning before password expires       : 7
➜  ~ sudo chage -M 10 test
➜  ~ sudo chage -l test   
Last password change                                    : Dec 05, 2017
Password expires                                        : Dec 15, 2017
Password inactive                                       : never
Account expires                                         : never
Minimum number of days between password change          : 0
Maximum number of days between password change          : 10
Number of days of warning before password expires       : 7
➜  ~ sudo chage -M -1 test
➜  ~ sudo chage -l test   
Last password change                                    : Dec 05, 2017
Password expires                                        : never
Password inactive                                       : never
Account expires                                         : never
Minimum number of days between password change          : 0
Maximum number of days between password change          : -1
Number of days of warning before password expires       : 7
```

**Comment by slash3b** [comment]
thanks a lot!

**Comment by sbrl** [comment]
Thanks, @slash3b! 😺

**Pull request merged**